Lib9c/Action/CreatePreviewNetAdminAccount.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Libplanet.Assets;
+using Nekoyume.Model.State;
+
+namespace Nekoyume.Action
+{
+    /// <summary>
+    // https://github.com/planetarium/lib9c/pull/1028
+    /// </summary>
+    [Serializable]
+    [ActionObsolete(BlockChain.Policy.BlockPolicySource.PreviewNetAdminAccountAllowIndex)]
+    [ActionType("create_preview_net_admin_account")]
+    public class CreatePreviewNetAdminAccount : ActionBase
+    {
+        public override IValue PlainValue =>
+            new Bencodex.Types.Dictionary(new Dictionary<IKey, IValue>
+            {
+            });
+
+        public override void LoadPlainValue(IValue plainValue)
+        {
+        }
+
+        public CreatePreviewNetAdminAccount()
+        {
+        }
+
+        public override IAccountStateDelta Execute(IActionContext context)
+        {
+            var states = context.PreviousStates;
+
+            CheckObsolete(BlockChain.Policy.BlockPolicySource.PreviewNetAdminAccountAllowIndex, context);
```

Comment:
I think we must check signer is admin like [this](https://github.com/planetarium/lib9c/blob/v100200/Lib9c/Action/PatchTableSheet.cs)